tests/bug-4759/test.yaml

```diff
@@ -0,0 +1,302 @@
+# *** Add configuration here ***
+
```

Comment:
Could you please remove these lines? They're a leftover from the createst script and should not go in the final tests.